taichi/transforms/simplify.cpp

```diff
@@ -1066,6 +1066,10 @@ class BasicBlockSimplify : public IRVisitor {
     return;
   }
 
+  void visit(ContinueStmt *stmt) override {
+    return;
+  }
+
   static bool is_global_write(Stmt *stmt) {
     return stmt->is<GlobalStoreStmt>() || stmt->is<AtomicOpStmt>();
```

Comment:
Btw, is `AtomicOpStmt` always about global variables?

Reply:
Before offloading it can also be used on locals. E.g.

```python
@ti.kernel
def test():
  a = 0
  for i in range(100): # parallel loop
    a += i # atomic add
```

Reply:
I think it's either SNode variables or global temporary variables. I forgot whether if it supports arg variables..

Reply:
I thought arg atomics is also supported...? What type of variables are not supported with atomic op?